tests/test_serialisation.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
 
 import equinox as eqx
+import equinox.serialisation
```

Comment:
It should be possible to access stuff just via `eqx`. (Or more broadly also `eqx.nn` and `eqx.experimental`.)